region/maridia/inner-green.json

```diff
@@ -2903,10 +3091,47 @@
                 {
                   "name": "Base",
                   "notable": false,
-                  "requires": []
+                  "requires": [ "f_ShaktoolDoneDigging" ]
+                },
+                {
+                  "name": "Vanilla",
+                  "notable": false,
+                  "requires": [ "h_ShaktoolVanillaFlag" ],
+                  "note": "Entering from the right door gives a weird alignment glitch, but it's possible to exit back through the door and that triggers the event flag, which opens up the room on the next re-entry"
+                },
+                {
+                  "name": "Reverse Shaktool",
+                  "notable": false,
+                  "requires": [ "h_canUsePowerBombs" ],
+                  "note": "Let the Snails (Yards) dig through the sand and follow them. They will not dig while off screen or while Samus is facing them, even while morphed."
+                },
+                {
+                  "name": "G-Mode Morph Reverse Shaktool",
+                  "notable": true,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [2],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "Bombs",
+                    {"or": [
+                      {"and": [
+                        "Gravity",
+                        "h_canArtificialMorphIBJ"
+                      ]},
+                      "SpringBall"
+                    ]},
+                    {"ammo": { "type": "PowerBomb", "count": 1 }}
+                  ],
+                  "note": [
+                    "The Snails will dig through the sand without showing any visual difference. Each block they pass through gets closer to overloading PLMs.",
+                    "In order to PB the left wall before PLMs are overloaded, it is necessary to kill one of the Snails with a bomb before it digs any sand.",
```

Comment:
Can the snails travel in the same path or will that count as both eating sand blocks?  They'll be in different positions if you use gimmobile. 

With springball you can stunlock one then move it off camera.